webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/ncf/TestEditorTechniqueWriter.scala

```diff
@@ -111,7 +112,7 @@ class TestEditorTechniqueWriter extends Specification with ContentMatchers with
 
   override def afterAll(): Unit = {
     if (java.lang.System.getProperty("tests.clean.tmp") != "false") {
-      FileUtils.deleteDirectory(new File(basePath))
+      // FileUtils.deleteDirectory(new File(basePath))
```

Comment:
todo

Reply:
You can pass that parameter through intellij to avoid that: 
![image](https://github.com/Normation/rudder/assets/44649/ee018f26-9405-4a69-8698-c915c458fa2c)
![image](https://github.com/Normation/rudder/assets/44649/b70145c2-a11b-4294-9ef4-c4d2b7a421f1)

You can even tells intellij to launch all you tests with that param (so that you don't have to edit each unit test):
![image](https://github.com/Normation/rudder/assets/44649/2d899b7e-be07-45c4-849e-f9aeed745f9b)